lib/ansible/modules/cloud/docker/docker_prune.py

```diff
@@ -105,6 +105,15 @@
     networks: yes
     volumes: yes
     builder_cache: yes
+
+- name: Prune everything (including non-dangling images)
+  docker_prune:
+    containers: yes
+    images: yes
+      dangling: false
```

Comment:
You need to put this into the `images_filters` option, otherwise it won't work (currently it is invalid YAML):
```suggestion
    images_filters:
      dangling: false
```
(That's also why the tests currently fail.)